snow/uptime/manager.go

```diff
@@ -68,33 +69,36 @@ func (m *manager) StartTracking(nodeIDs []ids.NodeID, subnetID ids.ID) error {
 
 		durationOffline := now.Sub(lastUpdated)
 		newUpDuration := upDuration + durationOffline
-		if err := m.state.SetUptime(nodeID, subnetID, newUpDuration, now); err != nil {
+		if err := m.state.SetUptime(nodeID, newUpDuration, now); err != nil {
 			return err
 		}
 	}
-	m.trackedSubnets.Add(subnetID)
+	m.startedTracking = true
 	return nil
 }
 
-func (m *manager) StopTracking(nodeIDs []ids.NodeID, subnetID ids.ID) error {
+func (m *manager) StopTracking(nodeIDs []ids.NodeID) error {
+	// TODO: this was not here before, should we add it?
+	if !m.startedTracking {
+		return nil
```

Comment:
If we are adding this handling - it seems like we should probably error here no? (Although I'm not sure the current code is correct if we were to do that)

Reply:
I added errors and checks with `StartedTracking` so we don't recall them twice. in case bootstrapping/shutdown is called multiple times + some test helpers.